<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>596</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Naziv">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Naziv</string>
   </property>
  </widget>
  <widget class="QLabel" name="Naziv_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Opis</string>
   </property>
  </widget>
  <widget class="QLabel" name="Naziv_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kategorija</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="opisText">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>541</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nazivText">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="kategorijeComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="obrisiUnoseGumb">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Obriši</string>
   </property>
  </widget>
  <widget class="QPushButton" name="unosGumb">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Unesi Pojam</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uneseniPojmoviGumb">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>410</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Uneseni Pojmovi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="obrisiPojamGumb">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>490</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Obriši Pojam</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
